catalog/app/utils/workflows.spec.js

```diff
@@ -36,14 +32,9 @@ describe('utils/workflows', () => {
         version: "1"
         workflows: []
       `
-      const config = workflows.parse(data)
-
+      const parser = () => workflows.parse(data)
       it('should return default empty values', () => {
```

Comment:
this description doesnt match the test code (it expects an error thrown, not a value returned)